app/components/ui-kit/switch.js

```diff
@@ -0,0 +1,67 @@
+import Component from '@ember/component';
+import { computed } from '@ember/object';
+import { reads } from '@ember/object/computed';
+import { checkDictionary } from 'travis/utils/ui-kit/assertions';
+
+// Size definitions
+const SIZES = {
+  MD: 'md',
+};
+const DEFAULT_SIZE = SIZES.MD;
+
+const DOT_SIZES = {
+  [SIZES.MD]: 4,
+};
+
+const SPACE_SIZES = {
+  [SIZES.MD]: 5,
+};
+
+// Color definitions
+const COLORS = {
+  BLUE: 'blue-300',
+};
+const DEFAULT_COLOR = COLORS.BLUE;
+
+const BG_COLORS = {
+  inactive: 'grey-300',
+};
+
+// Component definition
+export default Component.extend({
+
+  // Public interface
+  tagName: '',
```

Comment:
Seems like it should be outside of the public interface definition

Reply:
Yah totally, moved it :+1: